packages/rest/src/validation/request-body.validator.ts

```diff
@@ -156,10 +156,11 @@ export async function validateValueAgainstSchema(
   try {
     const validationResult = await validate(value);
     // When value is optional & values is empty / null, ajv returns null
-    if (validationResult || validationResult === null) {
-      debug(`Value from ${options.source} passed AJV validation.`);
-      return;
-    }
+    debug(
```

Comment:
nitpick: I remember checking the null value to pass some test, but if the check is not needed, please also remove the comment above.